shared/lib_utility_rate.cpp

```diff
@@ -13,9 +13,21 @@ UtilityRate::UtilityRate(
 	m_ecWeekday = ecWeekday;
 	m_ecWeekend = ecWeekend;
 	m_ecRatesMatrix = ecRatesMatrix;
-	m_ecRealTimeBuy = ecRealTimeBuy;
+	m_ecRealTimeBuy = std::move(ecRealTimeBuy);
 }
 
```

Comment:
Was this started off the develop branch or urdbjson branch? There will need to be several utility rate tests to determine if wae are going to use lib_utility_rate anywhere but for cmod_battery

Reply:
it was started off develop. the move is safe since the parameter is passed in by value so it has to be a copy of the original.

agreed we need utility rate tests, but i made very few modifications to utility rate except adding copy constructors